gamemode/modules/base/cl_jobmodels.lua

```diff
@@ -1,9 +1,32 @@
 -- Create a table for the preferred playermodels
+--
+-- Note: the server column wasn't always there, so players may not have a value
+-- for it, even though the code in this file always adds it. That is why it is
+-- added as a NULLable column.
 sql.Query([[CREATE TABLE IF NOT EXISTS darkp_playermodels(
     jobcmd VARCHAR(45) NOT NULL PRIMARY KEY,
-    model VARCHAR(140) NOT NULL
+    model VARCHAR(140) NOT NULL,
+    server TEXT NULL
```

Comment:
One thing I realized as well: with `jobcmd` still being the primary key, two different servers with the same `jobcmd` would not be able to have two rows in the table.